scheduler/src/cook/mesos/api.clj

```diff
@@ -1093,7 +1093,8 @@
   {:job [s/Uuid]})
 
 (def UpdateRetriesRequest
-  {(s/optional-key :jobs) [s/Uuid]
+  {(s/optional-key :job) s/Uuid
```

Comment:
@shamsimam 👍  done.  Added a unit test to prove that "jobs" in conjunction with "job" is considered malformed.